components/dashboard/TextArea/index.tsx

```diff
@@ -49,14 +50,33 @@ const TextArea = ({ shiftRight, setShiftRight }: TextAreaProps) => {
       ).withConverter(notesConverter)
   );
 
+  const addToStore1 = async (
+    key: string,
+    value: { editorTitle: string; editorContent: string }
+  ) => {
+    set(key, value);
+  };
+
+  const getFromStore1 = async (selectedNoteId: string) => {
+    if (!selectedNoteId) return;
+    return (
+      get(selectedNoteId) || {
```

Comment:
What if get returns null or undefined? I think we should be adding a check here.

Reply:
if it does, we return that empty object